python/lsst/ts/ATDome/config_schema.py

```diff
@@ -1,3 +1,29 @@
+# This file is part of ts_ATDome.
+#
+# Developed for the LSST Data Management System.
```

Comment:
Please update this and all other files to Vera C. Rubin Observatory.

Reply:
Good catch. I did that for all the packages.